app/api/v1/endpoints/postmark_webhook.py

```
@@ -1,8 +1,9 @@
 # =============================================================================
-# app/api/v1/endpoints/postmark_webhook.py
+# app/api/v1/endpoints/postmark_webhook.py (COMPLETE UPDATED VERSION)
 # =============================================================================
 from fastapi import APIRouter, Depends, HTTPException, status, Request
```

Ruff (0.11.9)

`fastapi.Request` imported but unused

Remove unused import: `fastapi.Request`

(F401)